src/epoch.rs

```
@@ -67,24 +64,24 @@ impl Epoch {
     /// let prev_prev = prev.prev();
     /// assert!(prev_prev < prev);
     /// ```
+    #[allow(clippy::precedence)]
```

Could you just add a parenthesis instead of suppressing the error?

---

Well thought of.